### ISSUE TITLE
Enhancement: AmazonSQS connector should support DefaultAWSCredentialsProviderChain

### ISSUE THREAD

**Issue by nsn** [payara/ecosystem-support#44]
# Description #
----------

`AmazonSQSActivationSpec` should support DefaultAWSCredentialsProviderChain, either by providing a dedicated `@ActionvationConfigProperty` or just by default:

```
return new DefaultAWSCredentialsProviderChain().getCredentials();
```
## Expected Outcome

The default credentials chain should be accessible for users

## Current Outcome

only env vras and hard-coded string literals are supported, which makes it difficult to manage credentials securely

## Context

All AWS SDKs support chaining of credential providers, by default checking for env vars, ~/.aws files and when deployed in Amazon containers obtaining credentials from the network. Since the AmazonSQS adapter needs to be deployed as a .rar it's impossible to implement not-yet-supported credentials providers, so support for default scheme can only be implemented in the library itself. This would take the burden of managing different secrets for different environments securely from the users.

## Ecosystem Tool

Cloud Connectors

**Comment by smillidge** [comment]
We have a PR for this https://github.com/payara/Cloud-Connectors/pull/379 so plan to provide something based on the PR soon.

**Comment by JamesHillyard** [comment]
Hi @nsn,

As Steve has said, there is currently a community contribution which we believe will implement the functionality you are requesting. When there is an update regarding this contribution, I will inform you.

Thanks,
James

**Comment by smillidge** [comment]
This was fixed by https://github.com/payara/Cloud-Connectors/pull/539

**Comment by JamesHillyard** [comment]
Hi @nsn,

As Steve recently said, there has been a PR to fix this. This PR will be included in the Cloud Connectors 0.8.0, which will be released soon. I will again update you when Cloud Connectors 0.8.0 is available.

Thanks,
James

**Comment by JamesHillyard** [comment]
Hi @nsn,

Cloud Connectors 0.8.0 has now been released and contains your suggested enhancement implemented in the PR mentioned by Steve, https://github.com/payara/Cloud-Connectors/pull/539.

Thanks,
James
